metron-analytics/metron-statistics/src/main/java/org/apache/metron/statistics/outlier/MedianAbsoluteDeviationFunctions.java

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
Agreed; given the structure, I think the profiler client itself should have a LRU cache that `PROFILE_GET` should pull from before hitting HBase.  I'll make the JIRA.